screens/WalletDetailScreen.tsx

```diff
@@ -197,6 +212,36 @@ const WalletDetailScreen = ({
       ? null
       : amountDecimal;
 
+  useEffect(() => {}, [
+    tokenBlacklist,
+    addTokenToBlackList,
+    removeTokenFromBlackList
+  ]);
+
+  const hideButton = (tokenId: string) => (
```

Comment:
Styling / placement of this toggle needs work.

The top of the screen is the most valuable real-estate, whereas this feature is pretty low priority the majority of the time.  
Sync with me for ideas, lots to consider where/how it's placed, especially in regards to target areas, proximity to other things, etc.